modules/wayfire/hm.nix

```
@@ -0,0 +1,65 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+{
+  options.stylix.targets.wayfire.enable =
+    config.lib.stylix.mkEnableTarget "wayfire" true;
+
+  config =
+    lib.mkIf (config.stylix.enable && config.stylix.targets.wayfire.enable)
+      (
+        let
+          inherit (config.lib.stylix) colors;
+          rgba = rgb: a: "\\#${rgb}${a}";
+          rgb = (lib.flip rgba) "ff";
```

I think `lib.flip` is more confusing than the following:

```suggestion
          rgb = rgb: rgba rgb "ff";
```